src/coreclr/nativeaot/Runtime/unix/UnixNativeCodeManager.cpp

```diff
@@ -525,6 +525,90 @@ int UnixNativeCodeManager::TrailingEpilogueInstructionsCount(PTR_VOID pvAddress)
         return -1;
     }
 
+#elif defined(TARGET_ARM64)
+
+// ldr with unsigned immediate
+// 1x11 1001 x1xx xxxx xxxx xxxx xxxx xxxx
+#define LDR_BITS1 0xB9400000
+#define LDR_MASK1 0xBF400000
+
+// ldr with pre/post/no offset
+// 1x11 1000 010x xxxx xxxx xxxx xxxx xxxx
+#define LDR_BITS2 0xB8400000
+#define LDR_MASK2 0xBFE00000
+
+// ldr with register offset
+// 1x11 1000 011x xxxx xxxx 10xx xxxx xxxx
+#define LDR_BITS3 0xB8600800
+#define LDR_MASK3 0xBFE00C00
+
+// ldp with signed offset
+// x010 1001 01xx xxxx xxxx xxxx xxxx xxxx
+#define LDP_BITS1 0x29400000
+#define LDP_MASK1 0x7FC00000
+
+// ldp with pre/post/no offset
+// x010 100x x1xx xxxx xxxx xxxx xxxx xxxx
+#define LDP_BITS2 0x28400000
```

Comment:
I will trust you that you got these magic numbers right :-)

Reply:
It took quite some time to construct and proof-read these.
The "BITS" parts for the individual instruction patterns ended up matching constants from instrarm64.h  
And that is encouraging :-)

https://github.com/dotnet/runtime/blob/2201016c1e13bdb9abf49e2e38cadf4ee0568df2/src/coreclr/jit/instrsarm64.h#L613